src/utils/parsers/languages.ts

```diff
@@ -0,0 +1,904 @@
+    vho: 'VHDL',
+    vhs: 'VHDL',
+    vht: 'VHDL',
+    vhw: 'VHDL',
+    vala: 'Vala',
+    vapi: 'Vala',
+    veo: 'Verilog',
+    vim: 'VimL',
+    vb: 'Visual Basic',
+    bas: 'Visual Basic',
+    frm: 'Visual Basic',
+    frx: 'Visual Basic',
+    vba: 'Visual Basic',
+    vbhtml: 'Visual Basic',
+    vbs: 'Visual Basic',
+    volt: 'Volt',
+    vue: 'Vue',
+    owl: 'Web Ontology Language',
+    webidl: 'WebIDL',
+    x10: 'X10',
+    xc: 'XC',
+    xml: 'XML',
+    ant: 'XML',
+    axml: 'XML',
+    ccxml: 'XML',
+    clixml: 'XML',
+    cproject: 'XML',
+    csl: 'XML',
+    csproj: 'XML',
+    ct: 'XML',
+    dita: 'XML',
+    ditamap: 'XML',
+    ditaval: 'XML',
+    'dll.config': 'XML',
+    dotsettings: 'XML',
+    filters: 'XML',
+    fsproj: 'XML',
+    fxml: 'XML',
+    glade: 'XML',
+    grxml: 'XML',
+    iml: 'XML',
+    ivy: 'XML',
+    jelly: 'XML',
+    jsproj: 'XML',
+    kml: 'XML',
+    launch: 'XML',
+    mdpolicy: 'XML',
+    mxml: 'XML',
+    nproj: 'XML',
+    nuspec: 'XML',
+    odd: 'XML',
+    osm: 'XML',
+    plist: 'XML',
+    props: 'XML',
+    ps1xml: 'XML',
+    psc1: 'XML',
+    pt: 'XML',
+    rdf: 'XML',
+    rss: 'XML',
+    scxml: 'XML',
+    srdf: 'XML',
+    storyboard: 'XML',
+    stTheme: 'XML',
+    'sublime-snippet': 'XML',
+    targets: 'XML',
+    tmCommand: 'XML',
+    tml: 'XML',
+    tmLanguage: 'XML',
+    tmPreferences: 'XML',
+    tmSnippet: 'XML',
+    tmTheme: 'XML',
+    ui: 'XML',
+    urdf: 'XML',
+    ux: 'XML',
+    vbproj: 'XML',
+    vcxproj: 'XML',
+    vssettings: 'XML',
+    vxml: 'XML',
+    wsdl: 'XML',
+    wsf: 'XML',
+    wxi: 'XML',
+    wxl: 'XML',
+    wxs: 'XML',
+    x3d: 'XML',
+    xacro: 'XML',
+    xaml: 'XML',
+    xib: 'XML',
+    xlf: 'XML',
+    xliff: 'XML',
+    xmi: 'XML',
+    'xml.dist': 'XML',
+    xproj: 'XML',
+    xsd: 'XML',
+    xul: 'XML',
+    zcml: 'XML',
+    'xsp-config': 'XPages',
+    'xsp.metadata': 'XPages',
+    xpl: 'XProc',
+    xproc: 'XProc',
+    xquery: 'XQuery',
+    xq: 'XQuery',
+    xql: 'XQuery',
+    xqm: 'XQuery',
+    xqy: 'XQuery',
+    xs: 'XS',
+    xslt: 'XSLT',
+    xsl: 'XSLT',
+    xojo_code: 'Xojo',
+    xojo_menu: 'Xojo',
+    xojo_report: 'Xojo',
+    xojo_script: 'Xojo',
+    xojo_toolbar: 'Xojo',
+    xojo_window: 'Xojo',
+    xtend: 'Xtend',
+    yml: 'YAML',
+    reek: 'YAML',
+    rviz: 'YAML',
+    'sublime-syntax': 'YAML',
+    syntax: 'YAML',
+    yaml: 'YAML',
+    'yaml-tmlanguage': 'YAML',
+    yang: 'YANG',
+    y: 'Yacc',
+    yacc: 'Yacc',
+    yy: 'Yacc',
+    zep: 'Zephir',
+    zimpl: 'Zimpl',
+    zmpl: 'Zimpl',
+    zpl: 'Zimpl',
+    desktop: 'desktop',
+    'desktop.in': 'desktop',
+    ec: 'eC',
+    eh: 'eC',
+    edn: 'edn',
+    fish: 'fish',
+    mu: 'mupad',
+    nc: 'nesC',
+    ooc: 'ooc',
+    rst: 'reStructuredText',
+    rest: 'reStructuredText',
+    'rest.txt': 'reStructuredText',
+    'rst.txt': 'reStructuredText',
+    wisp: 'wisp',
+    prg: 'xBase',
+    prw: 'xBase'
+  };
```

Comment:
is this really necessary for bloom? and also in this size?

Reply:
For more accurate responses, yes unfortunately. The size could be stripped down once I find out which languages bloom does support.